src/scipp/logging.py

```diff
@@ -140,7 +139,6 @@ def display_logs() -> None:
             'Cannot display log widgets because no widget handler is installed.')
 
     from IPython.display import display
-    inject_style()
```

Comment:
Now you need to add the style to the widget's HTML.

Reply:
I thought since it will be inline in the variable repr it should be sufficient?

Reply:
Only once you show a variable. If a user displays a log widget before outputting any variables, tables, svg, the style is missing and formatting is completely of:
![log-widget-missing-style](https://user-images.githubusercontent.com/11393224/141480214-1c860667-5d56-45f9-876a-3004522e39e3.png)

Reply:
Oh, I didn't realize that the CSS is not just for the scipp object repr, but also the logger itself!